core/src/main/kotlin/Samples/KotlinWebsiteSampleProcessingService.kt

```diff
@@ -165,7 +165,7 @@ open class KotlinWebsiteSampleProcessingService
         return sampleBuilder.text
     }
 
-    val importsToIgnore = arrayOf("samples.*").map { ImportPath.fromString(it) }
```

Comment:
LGTM. Note however, that there are some other declarations in the `samples` package, that could leak into the generated docs. Thus I'd prefer if the samples kept using a star-import of the `samples` package and dokka reported a warning otherwise (in case of non-star import).